tests/logging_tests/tests.py

```diff
@@ -513,3 +516,60 @@ def test_extra_args(self):
                 }},
             )]
         )
+
+
+class LogFormattersTests(SetupDefaultLoggingMixin, SimpleTestCase):
+    def test_server_formatter_styles(self):
```

Comment:
I guess most test classes have a blank line after the class and before the first test method.